webapp/lib/charts.dart

```diff
@@ -176,6 +176,8 @@ class DailyTimeseriesLineChartView {
     if (timeScaleUnit == 'hour') {
       chart.options.scales.xAxes[0].time = (new chartjs.TimeScale()
                                               ..displayFormats = new chartjs.TimeDisplayFormat(hour: 'D/MM hA'));
+      chart.options.scales.xAxes[0].type = 'time';
+      chart.options.scales.xAxes[0].ticks.max = xUpperLimit?.toIso8601String();
```

Comment:
I think this is missing the lower limit?

```suggestion
      chart.options.scales.xAxes[0].ticks.min = xLowerLimit?.toIso8601String();
      chart.options.scales.xAxes[0].ticks.max = xUpperLimit?.toIso8601String();
```